bin/cml/runner.js

```diff
@@ -1,589 +1,13 @@
-    }, 1000);
-  }
-
-  if (!noRetry) {
-    if (cml.driver === 'github') {
-      const watcherSeventyTwo = setInterval(() => {
-        RUNNER_JOBS_RUNNING.forEach((job) => {
-          if (
-            new Date().getTime() - new Date(job.date).getTime() >
-            GH_5_MIN_TIMEOUT
-          ) {
-            shutdown({ ...opts, reason: 'timeout:35days' });
-            clearInterval(watcherSeventyTwo);
-          }
-        });
-      }, 60 * 1000);
-    }
-  }
-};
-
-const run = async (opts) => {
-  opts.workdir = opts.workdir || `${homedir()}/.cml/${opts.name}`;
-
-  process.on('unhandledRejection', (reason) =>
-    shutdown({ ...opts, error: new Error(reason) })
-  );
-  process.on('uncaughtException', (error) => shutdown({ ...opts, error }));
-
-  ['SIGTERM', 'SIGINT', 'SIGQUIT'].forEach((signal) => {
-    process.on(signal, () => shutdown({ ...opts, reason: signal }));
-  });
-
-  const {
-    driver,
-    workdir,
-    cloud,
-    labels,
-    name,
-    reuse,
-    reuseIdle,
-    dockerVolumes
-  } = opts;
-
-  await cml.repoTokenCheck();
-
-  const runners = await cml.runners();
-  const runner = await cml.runnerByName({ name, runners });
-  if (runner) {
-    if (!reuse)
-      throw new Error(
-        `Runner name ${name} is already in use. Please change the name or terminate the existing runner.`
-      );
-    winston.info(`Reusing existing runner named ${name}...`);
-    return;
-  }
-
-  if (
-    reuse &&
-    (await cml.runnersByLabels({ labels, runners })).find(
-      (runner) => runner.online
-    )
-  ) {
-    winston.info(
-      `Reusing existing online runners with the ${labels} labels...`
-    );
-    return;
-  }
-
-  if (reuseIdle) {
-    if (driver === 'bitbucket') {
-      throw new Error(
-        'cml runner flag --reuse-idle is unsupported by bitbucket'
-      );
-    }
-    winston.info(
-      `Checking for existing idle runner matching labels: ${labels}.`
-    );
-    const currentRunners = await cml.runnersByLabels({ labels, runners });
-    const availableRunner = currentRunners.find(
-      (runner) => runner.online && !runner.busy
-    );
-    if (availableRunner) {
-      winston.info('Found matching idle runner.', availableRunner);
-      return;
-    }
-  }
-
-  if (dockerVolumes.length && cml.driver !== 'gitlab')
-    winston.warn('Parameters --docker-volumes is only supported in gitlab');
-
-  if (driver === 'github')
-    winston.warn(
-      'Github Actions timeout has been updated from 72h to 35 days. Update your workflow accordingly to be able to restart it automatically.'
-    );
-
-  if (RUNNER_NAME)
-    winston.warn(
-      'ignoring RUNNER_NAME environment variable, use CML_RUNNER_NAME or --name instead'
-    );
-
-  winston.info(`Preparing workdir ${workdir}...`);
-  await fs.mkdir(workdir, { recursive: true });
-  await fs.chmod(workdir, '766');
-
-  if (cloud) await runCloud(opts);
-  else await runLocal(opts);
-};
+const { options, handler } = require('./runner/start');
```

Comment:
option of `--cloud-aws-subnet-id` printed twice? 
![image](https://user-images.githubusercontent.com/1105843/185987943-a59b357d-b278-4641-94d6-1e87963b6aae.png)

Reply:
`cml runner start --help`